docs/contributors_guide/maintenance/iso_release_procedures/Pre_Release_Checklist.rst

```diff
@@ -68,18 +53,27 @@ the RPMs for those projects exist by executing the ``pkg:check_published`` Rake
      Found Existing Remote RPM: pupmod-simp-sudosh-6.0.1-0.noarch.rpm
      ...
 
-   .. Important::
+   .. IMPORTANT::
 
       If you see a message like
       ``Warning:  Unable to generate build-specific YUM cache``, your
-      results are invalid, as connection to `packagecloud`_ failed.
+      results are invalid, as connection to `PackageCloud`_ failed.
+
+#. Manually verify the appropriate ``simp-doc`` RPM exists at `PackageCloud`_.
+
 
-#. Manually verify the appropriate ``simp-doc`` RPM exists at `packagecloud`_.
+For the external vendor RPMs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* Upload all vendor RPMs to the ``VERSION_Dependencies`` repository in
+  `PackageCloud`_. Any existing RPMs will not be overwritten.
 
-For the external verndor RPMs and OS RPMs:
+  * ``package_cloud push simp-project/VERSION_Dependencies/el/OS_MAJOR_VERSION /path/to/packages``
 
-* FILL-ME-IN
+.. WARNING::
+
+   **DO NOT** push any Operating System RPMs up to PackageCloud, those should
```

Comment:
Nothing from base or updates, but epel and other third-party repos are ok?

Reply:
Yes